modules/covid_vaccine/app/services/covid_vaccine/v0/registration_service.rb

```diff
@@ -45,6 +47,21 @@ def submit(attributes)
         CovidVaccine::V0::VetextService.new.put_vaccine_registry(attributes)
       end
 
+      def audit_log(attributes, user_type)
```

Comment:
Maybe there's a more elegant way of doing this but I wanted to be explicit about what attrs were being logged and booleanized.

Reply:
you could use Hash#slice but it looks like you're casting some of them boolean so I think what you're doing makes more sense and is more readable.